src/transposition.cpp

```diff
@@ -161,6 +161,19 @@ data_PITCHNAME TransPitch::GetPitchName()
 {
     return static_cast<data_PITCHNAME>(m_pname + PITCHNAME_c);
 }
+
+std::wstring TransPitch::GetPitchString()
+{
+    char pitchLetter = (m_pname + ('C' - 'A')) % 7 + 'A';
+    switch (m_accid) {
+        case -2: return std::wstring({ pitchLetter, L'𝄫' });
+        case -1: return std::wstring({ pitchLetter, L'♭' });
+        default: LogError("Transposition: Could not get Accidental for %i", m_accid);
+        case 0: return std::wstring({ pitchLetter });
+        case 1: return std::wstring({ pitchLetter, L'♯' });
+        case 2: return std::wstring({ pitchLetter, L'♯', L'♯' });
```

Comment:
When I tried to use the `𝄪` character, it would come out as `턪`.

Reply:
That most likely means that it would need to be added to the verovio text font in order to be viewed as a double sharp.

Reply:
But you are printing as text to standard output (or Javascript console), in which case it will more depend on the character encoding of the device, which is often now UTF-8 but not always.

Here are the encodings of double-sharp:
https://www.fileformat.info/info/unicode/char/1d12a/index.htm
Maybe using one of these might work.  Otherwise, I would just print "double-sharp" instead of the symbol since it is likely to get messed up regardless 😛

Reply:
I removed the code that printed this out when there was an error. (It wasn't able to print the wstring, and `.c_str()` got rid of the wide characters, so it wasn't useful).